plugins/modules/postgresql_user.py

```diff
@@ -920,6 +950,56 @@ def add_comment(cursor, user, comment, check_mode):
         return False
 
 
+def user_configuration(cursor, module, user, configuration):
+    """Updates the user's configuration parameters if necessary."""
+    current_config_query = "SELECT rolconfig FROM pg_roles WHERE rolname = %(user)s;"
+    cursor.execute(current_config_query, {"user": user})
+    current_config = cursor.fetchone()
+    current_config_dict = {}
+    changed = False
+
+    if current_config is None:
+        module.fail_json(msg="Can't find user %(user)s in 'pg_roles'" % {"user": user})
+
+    if current_config['rolconfig'] is not None:
+        for item in current_config['rolconfig']:
+            tokens = item.split("=", 1)
+            current_config_dict[tokens[0]] = tokens[1]
```

Comment:
```suggestion
    if current_config['rolconfig'] is not None:
        for item in current_config['rolconfig']:
            tokens = item.split("=", 1)
            current_config_dict[tokens[0]] = tokens[1]
```
i would suggest making it a dict as mentioned above and move this to a separate function. You could cover it with unit tests then if you wish, though only integration tests are mandatory.